cypress/integration/taskQueue/route.ts

```diff
@@ -60,13 +62,23 @@ describe("Task Queue", () => {
     cy.get(".ant-table-row").should("have.length", 0);
   });
 
-  it("Scrolls to current task if taskId param in url", () => {
-    cy.visit(
-      "/task-queue/osx-108/evergreen_lint_lint_service_patch_5e823e1f28baeaa22ae00823d83e03082cd148ab_5e4ff3abe3c3317e352062e4_20_02_21_15_13_48"
-    );
-
-    cy.get(".ant-table-row-selected").contains("13").should("be.visible");
-  });
+  it(
+    "Scrolls to current task if taskId param in url",
+    { scrollBehavior: false },
+    () => {
+      cy.intercept("POST", GQL_URL, (req) => {
+        aliasQuery(req, "DistroTaskQueue");
+      });
+      cy.viewport(1920, 1080);
```

Comment:
Should we set the viewport globally in a config file somewhere? I notice we call this in `host_filtering.ts` as well

Reply:
Yup i'm in support of setting a global viewport size instead of one off for tests